src/components/application_manager/include/application_manager/request_controller.h

```diff
@@ -205,6 +204,17 @@ class RequestController {
                             const uint32_t& mobile_correlation_id,
                             const uint32_t& new_timeout);
 
+  /**
+   * @brief IsUpdateRequestTimeoutRequired check is update timeout required.
+   * @param app_id Connection key of application
+   * @param correlation_id Correlation ID of the mobile request
+   * @param new_timeout New timeout to be set in milliseconds
+   * @return true if the new timeout value is greater than the time remaining
+   * from the current timeout, otherwise - false
+   */
+  bool IsUpdateRequestTimeoutRequired(const uint32_t& app_id,
```

Comment:
@LitvinenkoIra can be const?

Reply:
@LuxoftAKutsan no because we call non const function inside 
https://github.com/smartdevicelink/sdl_core/pull/2748/files/501fe87a6ee8ff6ec63e214fdaded439f0a8169c#diff-da657a2dd4bcbd330cb238e5ffff2c42R158